source/io/JavaScriptInvoke.cpp

```diff
@@ -63,7 +87,10 @@ VIREO_EXPORT void* JavaScriptInvoke_GetParameterPointer(StaticTypeAndData *param
         // We a have pointer to a StringRef, we just need the StringRef.
         // So we can use functions that already work with StringRef on the JavaScript side.
         pData = *(StringRef*)pData;
+    } else if (parameterType->IsArray()) {
+        pData = *(TypedArrayCoreRef*)pData;
     }
+
```

Comment:
Similar to the other EggShell_GetPointer we should validate supported cases and return null otherwise